sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -46,6 +46,18 @@ public ConfidentialLedgerClient(Uri ledgerUri, TokenCredential credential) : thi
 
         /// <summary> The constitution is a script that assesses and applies proposals from consortium members. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call GetConstitutionAsync.
```

Comment:
Could we add "and parse the result" for cases that we parse a JSON body out of the response?

Reply:
done